packages/auth/src/access/model.ts

```diff
@@ -28,3 +32,15 @@ export class Access extends BaseModel {
   public identifier?: string
   public limits?: LimitData
 }
+
+export function toOpenPaymentsAccess(
+  accessItem: Access
+): OpenPaymentsAccessItem {
+  return {
+    actions: accessItem.actions,
+    identifier: accessItem.identifier ?? undefined,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    type: accessItem.type as any,
```

Comment:
This was a bit of a tough one. Without `any` it complains about including `identifier`, since it only exists on the `outgoing-payments` access item type:
<img width="514" alt="Screenshot 2023-02-02 at 7 44 54 PM" src="https://user-images.githubusercontent.com/15069181/216421650-c44e2cf3-4719-4be8-a4e1-47a17265285f.png">

By checking whether or not identifier exists, it complains about `limits`:

<img width="449" alt="Screenshot 2023-02-02 at 7 49 02 PM" src="https://user-images.githubusercontent.com/15069181/216421566-8d8a3dab-6ae0-4bdc-8202-edb26f2dedde.png">
<img width="510" alt="Screenshot 2023-02-02 at 7 49 42 PM" src="https://user-images.githubusercontent.com/15069181/216421577-0d186c52-ea00-4544-993b-74d4b2d597d0.png">

Since we don't need to do any operations after `toOpenPaymentsAccess`, I left it as `any`.